install/kubernetes/cilium/values.yaml

```diff
@@ -1332,6 +1332,10 @@ monitor:
   # endpoints filtering
   # serviceTopology: false
 
+  # -- Enable service load balancing via envoy proxy
```

Comment:
so, like, this enables the _option_ but not for all services by default, right? Can we clarify this?

When would we ever want this disabled?